### ISSUE TITLE
Softbutton Delay

### ISSUE THREAD

**Issue by rbright55** [smartdevicelink/sdl_ios#1778]
### Bug Report 
With the latest sdk (6.7.0), softbuttons are not updating as quickly. I believe it may be due to the addition of the softbutton manager.

##### Reproduction Steps
1.  Load app and update softbutton text every couple seconds using the screenmanager

##### Expected Behavior
Softbuttons should update on SYNC with the new text

##### Observed Behavior
Softbuttons are not updating as fast. ( issue was not seen sdk 6.5.0)

##### OS & Version Information
* iOS Version: 13.5
* SDL iOS Version: 6.7.0
* Testing Against: App Partner beta app
* SYNC 3

**Comment by theresalech** [comment]
Hi @rbright55, thank you for reporting this issue. We have already completed development and started testing for the iOS 7.0 release, but will look to include a fix for this issue in the following release. If you are able to submit a PR to resolve this issue, please let us know once that is ready for review. Thanks!

**Comment by joeljfischer** [comment]
Hi @rbright55, I have noticed some of this slowdown on TDKs, however I have not noticed this slowdown on Manticore, which leads me to believe that at least part of the issue is Sync related. I can update the internal queues to run on the highest possible QOS (and I do think that is a good idea), but unless you have some data that shows the internal queues clogging up and not sending the Shows, I'm not sure what else to do.

I have noticed that when Show requests are sent in too quick of a sequence, Sync will often slow down and not respond for a significant period of time.
